experimentation/tools/scripts/GetKeyAndDescription.java

```diff
@@ -0,0 +1,51 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.Set;
+
+import com.google.gson.JsonArray;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonPrimitive;
+import spoon.Launcher;
+import spoon.compiler.Environment;
+import spoon.compiler.SpoonResource;
+import spoon.compiler.SpoonResourceHelper;
+import spoon.reflect.CtModel;
+import spoon.reflect.code.CtLiteral;
+import spoon.reflect.declaration.CtPackage;
+import spoon.reflect.declaration.CtType;
+
+public class GetKeyAndDescription {
+
+    public static void main(String[] args) throws FileNotFoundException {
+        CtPackage processorPackage = getProcessorPackage(new File(args[0]));
+        Set<CtType<?>> allProcessors = processorPackage.getTypes();
+        JsonArray rules = new JsonArray();
+        allProcessors.stream().forEach(processor -> {
+            // Skips SoraldAbstractProcessor
+            if (processor.isAbstract()) return;
+
+            JsonObject rule = new JsonObject();
+
+            // Remove 'S' prefix from rule key
+            CtLiteral<String> ruleKey = processor.getAnnotations().stream()
+                    .filter(annotation -> annotation.getType().getSimpleName().equals("ProcessorAnnotation"))
+                    .findFirst().get().getValue("key");
```

Comment:
A shorter way to find annotation would have been `getAnnotation(ProcessorAnnotation.class)` but this requires us to add sorald jar in the classpath too. I didn't want to edit the currently generated `cp.txt` so I went with this approach.